internal/providers/terraform/aws/testdata/data_transfer_test/data_transfer_test.golden

```diff
@@ -1,146 +1,207 @@
+ ├─ Outbound data transfer to Internet (over 150TB)        3,400  GB         $170.00 
+ └─ Outbound data transfer to other regions                  750  GB          $15.00 
+                                                                                     
+ aws_data_transfer.eu-west-3                                                         
+ ├─ Intra-region data transfer                             1,000  GB          $10.00 
+ ├─ Outbound data transfer to Internet (first 10TB)       10,240  GB         $921.60 
+ ├─ Outbound data transfer to Internet (next 40TB)        40,960  GB       $3,481.60 
+ ├─ Outbound data transfer to Internet (next 100TB)      102,400  GB       $7,168.00 
+ ├─ Outbound data transfer to Internet (over 150TB)        3,400  GB         $170.00 
+ └─ Outbound data transfer to other regions                  750  GB          $15.00 
+                                                                                     
+ aws_data_transfer.me-south-1                                                        
+ ├─ Intra-region data transfer                             1,000  GB          $10.00 
+ ├─ Outbound data transfer to Internet (first 10TB)       10,240  GB       $1,198.08 
+ ├─ Outbound data transfer to Internet (next 40TB)        40,960  GB       $4,526.08 
+ ├─ Outbound data transfer to Internet (next 100TB)      102,400  GB       $9,318.40 
+ ├─ Outbound data transfer to Internet (over 150TB)        3,400  GB         $221.00 
+ └─ Outbound data transfer to other regions                  750  GB          $82.87 
+                                                                                     
+ aws_data_transfer.sa-east-1                                                         
+ ├─ Intra-region data transfer                             1,000  GB          $10.00 
+ ├─ Outbound data transfer to Internet (first 10TB)       10,240  GB       $1,536.00 
+ ├─ Outbound data transfer to Internet (next 40TB)        40,960  GB       $5,652.48 
+ ├─ Outbound data transfer to Internet (next 100TB)      102,400  GB      $12,902.40 
+ ├─ Outbound data transfer to Internet (over 150TB)        3,400  GB         $387.60 
+ └─ Outbound data transfer to other regions                  750  GB         $103.50 
+                                                                                     
+ aws_data_transfer.us-east-1                                                         
+ ├─ Intra-region data transfer                             1,000  GB          $10.00 
+ ├─ Outbound data transfer to Internet (first 10TB)       10,240  GB         $921.60 
+ ├─ Outbound data transfer to Internet (next 40TB)        40,960  GB       $3,481.60 
+ ├─ Outbound data transfer to Internet (next 100TB)        5,800  GB         $406.00 
+ ├─ Outbound data transfer to US East regions                500  GB           $5.00 
+ └─ Outbound data transfer to other regions                  750  GB          $15.00 
```

Comment:
I compared these numbers to https://calculator.aws/#/createCalculator/DataTransfer and the `Intra-region data transfer` one does not match, AWS says this:
```
Intra region:
(1000 GB x 0.01 USD per GB outbound) + (1000 GB x 0.01 USD per GB inbound) = 20.00 USD 
```

The AWS EC2 pricing page mentions:
```
 Data Transfer within the same AWS Region

Data transferred "in" to and "out" from Amazon EC2, Amazon RDS, Amazon Redshift, Amazon DynamoDB Accelerator (DAX), and Amazon ElastiCache instances, Elastic Network Interfaces or VPC Peering connections across Availability Zones in the same AWS Region is charged at $0.01/GB in each direction.

IPv4: Data transferred “in” to and “out” from public or Elastic IPv4 address is charged at $0.01/GB in each direction.
IPv6: Data transferred “in” to and “out” from an IPv6 address in a different VPC is charged at $0.01/GB in each direction.
```

So I guess we have to double the number in our calcs?

Reply:
@alikhajeh1 Multiplied the quantity by 2 and updated the usage example. Anything else to change?

Reply:
Love it - thank you!